src/Layer/PointCloudLayer.js

```diff
@@ -231,6 +232,29 @@ class PointCloudLayer extends GeometryLayer {
         this.root = undefined;
     }
 
+    setRootBbox(min, max) {
```

Comment:
could you the same signature and this
`setRootBboxFromArray(bounds)`

Reply:
If i understand you, you wnat to add another methode, with a different signature. But currently this function is call 4 times, if we add a 2nd one, both will be called 2 times, won't it be unfactorizing some code ?